src/awsexplorer/regionNode.ts

```diff
@@ -61,6 +62,8 @@ export class RegionNode extends AWSTreeNodeBase {
                 serviceId: 's3',
                 createFn: () => new S3Node(ext.toolkitClientBuilder.createS3Client(this.regionCode)),
             },
+            { serviceId: 'ecs', createFn: () => new EcsNode(ext.toolkitClientBuilder.createEcsClient(this.regionCode))
```

Comment:
Move to line 59 for the sake of alphabetical ordering. While you're at it, move CloudWatch Logs as well? (we alphabetized that as `logs` but it looks bad)